src/templates/globals/util/handleSkipLink.js

```diff
@@ -0,0 +1,23 @@
+export const handleSkipLink = (e) => {
```

Comment:
i think i'd prefer this inside `@lib/utils/` instead of `templates/globals/util` (mostly because I want to nuke this globals directory (see: #191)), but open to reasons why it should stay here

Reply:
moved to utils